File: ResourceManager/ViewModels/Tools/ToolsPageViewModel.vb
Imports VBNetCore.Mvvm.Abstractions
Imports VBNetCore.Security
Imports ResourceManager.Configuration
Imports ResourceManager.Helpers
Imports System.Collections.ObjectModel
Imports System.Linq
Imports Microsoft.Extensions.DependencyInjection

Namespace ResourceManager.ViewModels
    Public Class ToolsPageViewModel
        Inherits PageViewModel
        Public Sub New()

            PageTitle = "Tools"
            PageIcon = AssetManager.Instance.GetImage("Tools.png")
            PageColor = ToolsPageColor
            MyBase.HasPermission = AuthManager.User.IsInPermission(VIEW_TOOLS_PERMISSION)

            MenuItems = New ObservableCollection(Of PageContentViewModel) From {
    Application.ServiceProvider.GetRequiredService(Of DatabaseContentViewModel)()
}
            SelectedItem = MenuItems.FirstOrDefault()
        End Sub
    End Class
End Namespace

File: ResourceManager/ViewModels/Tools/ToolsPageViewModel.cs
using Microsoft.Extensions.DependencyInjection;
using NetCore.Mvvm.Abstractions;
using NetCore.Security;
using ResourceManager.Configuration;
using ResourceManager.Helpers;
using System;
using System.Collections.ObjectModel;
using System.Linq;

namespace ResourceManager.ViewModels
{
    public class ToolsPageViewModel : PageViewModel
    {
        public ToolsPageViewModel()
        {

            PageTitle = "Tools";
            PageIcon = AssetManager.Instance.GetImage("Tools.png");
            PageColor = Constants.ToolsPageColor;
            HasPermission = AuthManager.User.IsInPermission(Constants.VIEW_TOOLS_PERMISSION);

            MenuItems = new ObservableCollection<PageContentViewModel>
            {
                App.ServiceProvider.GetRequiredService<DatabaseContentViewModel>()
            };
            SelectedItem = MenuItems.FirstOrDefault();
        }
    }
}
